<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>BDDGui</class>
 <widget class="QMainWindow" name="BDDGui">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>681</width>
    <height>509</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>BDD Solver</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QLabel" name="label_2">
          <property name="font">
           <font>
            <family>Yu Gothic UI</family>
            <pointsize>14</pointsize>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">color: rgb(55, 115, 171);</string>
          </property>
          <property name="text">
           <string>Give the number of vars: </string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="numOfVarsEdit">
          <property name="font">
           <font>
            <pointsize>14</pointsize>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">color: rgb(38, 78, 117);</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QLabel" name="label_5">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="font">
         <font>
          <family>Yu Gothic UI</family>
          <pointsize>11</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Give the number of variables the function will have </string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="Line" name="line">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_4">
        <item>
         <widget class="QLabel" name="label_3">
          <property name="font">
           <font>
            <family>Yu Gothic UI</family>
            <pointsize>14</pointsize>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">color: rgb(55, 115, 171);</string>
          </property>
          <property name="text">
           <string>Give the order of vars: </string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="orderOfVarsEdit">
          <property name="font">
           <font>
            <pointsize>14</pointsize>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">color: rgb(38, 78, 117);</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QLabel" name="label_6">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="font">
         <font>
          <family>Yu Gothic UI</family>
          <pointsize>12</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Example: (x &lt; y &lt; z) for 3 variables</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="Line" name="line_2">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_5">
        <item>
         <widget class="QLabel" name="label_4">
          <property name="font">
           <font>
            <family>Yu Gothic UI</family>
            <pointsize>14</pointsize>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">color: rgb(55, 115, 171);</string>
          </property>
          <property name="text">
           <string>Give the function: </string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="functionEdit">
          <property name="font">
           <font>
            <pointsize>14</pointsize>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">color: rgb(38, 78, 117);</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QLabel" name="label_7">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="font">
         <font>
          <family>Yu Gothic UI</family>
          <pointsize>12</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Example: (x + y' + z) for 3 variables</string>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <widget class="Line" name="line_3">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_6">
          <item>
           <widget class="QLabel" name="problemChecker">
            <property name="font">
             <font>
              <family>Yu Gothic UI</family>
              <pointsize>11</pointsize>
             </font>
            </property>
            <property name="text">
             <string>Problem</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <spacer name="verticalSpacer">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QCommandLinkButton" name="solveButton">
          <property name="text">
           <string>Solve</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCommandLinkButton" name="clearButton">
          <property name="text">
           <string>Clear</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
    <item>
     <widget class="Line" name="line_4">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <item>
       <widget class="QLabel" name="bddTreeView">
        <property name="font">
         <font>
          <family>Yu Gothic UI</family>
          <pointsize>14</pointsize>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">color: rgb(55, 115, 171);</string>
        </property>
        <property name="text">
         <string>Here your bdd result will appear.</string>
        </property>
        <property name="margin">
         <number>20</number>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources>
  <include location="../../../../../../Desktop/MySourse.qrc"/>
 </resources>
 <connections/>
</ui>
